src/pages/deploying-explorer/railway.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: Deploying Explorer on Railway
+short: Railway
+---
+import { Steps } from "nextra/components";
+
+# Deploying Explorer on Railway
+
+Explorer can be deployed as a Railway service using the `am-proxy` Docker image. [Railway now supports deployinj the services from Docker images directly](https://railway.app/changelog/2023-09-01-cron-jobs-ui#docker-image-registry-deploys-ga).
+
+<Steps>
+
+### Create a Railway service from a Docker image
+
+Make sure to provide the correct Prometheus URL (can be internal Railway URL) to the `PROMETHEUS_URL` environment variable and kick off the deployment.
```

Comment:
Maybe also mentioning here the Variable PORT and LISTEN_ADDRESS ? For my setup there where needed.

Reply:
thanks added!